src/main/java/org/opensearch/security/rest/TenantInfoAction.java

```diff
@@ -65,12 +65,15 @@
 import org.opensearch.threadpool.ThreadPool;
 
 import com.google.common.collect.ImmutableList;
+import static org.opensearch.security.dlic.rest.support.Utils.addRoutesPrefix;
 
 public class TenantInfoAction extends BaseRestHandler {
-    private static final List<Route> routes = ImmutableList.of(
-            new Route(GET, "/_opendistro/_security/tenantinfo"),
-            new Route(POST, "/_opendistro/_security/tenantinfo")
-    );
+    private static final List<Route> routes = addRoutesPrefix(
+            ImmutableList.of(
+                new Route(GET, "/tenantinfo"),
+                new Route(POST, "/tenantinfo")
+            ),
+            "/_opendistro/_security", "/_plugins/_security");
```

Comment:
Should use the same overriden method as before `addRoutesPrefix(List<Route)`

Reply:
For this one we do not have `/api/` in the routes so we should provide the customized paths.